ScanToShop/Managers/DatabaseManager.h

```diff
@@ -0,0 +1,22 @@
+//
+//  DatabaseManager.h
+//  ScanToShop
+//
+//  Created by Joey R. Hernandez Perez on 7/17/20.
+//  Copyright © 2020 joeyrhernandez1. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "User.h"
+#import <Parse/Parse.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface DatabaseManager : NSObject
+
++(void)saveUser:(User *)user withCompletion:(void(^)(BOOL success, NSError *error))completion;
```

Comment:
This is much better. Probably don't need both error and success but feel free to change in next pr.